.eslintrc

```diff
@@ -5,5 +5,13 @@
     "react": {
       "version": "detect"
     }
-  }
+  },
+  "overrides": [
+    {
+      "files": ["src/components/TabNavigator.js", "*.stories.js"],
```

Comment:
This makes sense to me for the glob pattern type overrides like "All stories", but for one off/specific files, could we go with inline overrides in the independant files? I feel that's more readable and discoverable from a dev point of view.

So you'd add something like this to the top of `TabNavigator.js`
```js
/* eslint-disable react/prop-types */
```

I also recommend adding a comment as to why it's being overriden. I've done this a couple times when wrapping an existing component. So something like:

```js
/* eslint-disable react/prop-types -- The @react-navigation/bottom-tabs handles prop types for us */
```
(Assuming that's why you're disabling it)

Docs: https://eslint.org/docs/user-guide/configuring.html#disabling-rules-with-inline-comments

Reply:
Oh I didn't realize it would disable for the whole file. I thought you had to put it above each line. I've added it because it's throwing `error  'color' is missing in props validation` for all the `<Label />` components such as [this one](https://github.com/bcgov/parks-adventure-mobile/pull/15/files#diff-57b9b9b875cdc3f095395c48932301d5R24).

Reply:
Yeah there's the `eslint-disable-next-line` if you just want to disable one line. That reasoning makes sense to me to override it for this component.